### PR TITLE
code review for metric util

### PR DESCRIPTION
**Pull request by jananjoy**
Description: Code review branch for metric util file 
I have only  minor changes and reformatting some comments.
Clarification and discussion will be  written  in the code comments.

### CODE REVIEW

File: pulsar_metrics/metrics/utils.py [hunk 1]
```diff
@@ -34,36 +34,35 @@ def get_population_percentages(new: pd.Series, reference: pd.Series, binned: boo
 
             return percents
     except Exception as e:
-        print(f"Error while calculating population percentages: {str(e)}")
+        print(f"Error in get_population_percentages() while calculating population percentages: {str(e)}")
```
Comment (jananjoy) [inline pulsar_metrics/metrics/utils.py:37]: Comment: if is_numeric_dtype(new) & is_numeric_dtype(reference): If this condition returns false. (generally return true or false) what is the execution work flow ?

File: pulsar_metrics/metrics/utils.py [hunk 2]
```diff
@@ -34,36 +34,35 @@ def get_population_percentages(new: pd.Series, reference: pd.Series, binned: boo
...
 
 
 def kullback_leibler_divergence(new: pd.Series, reference: pd.Series, binned: bool = False):
     """
-    Calculates the Kullback-Leibler divergence
+    Calculate Kullback-Leibler divergence (KL divergence) for samples (new,reference)
     """
 
     percents = get_population_percentages(new, reference, binned)
```
Comment (jananjoy) [inline pulsar_metrics/metrics/utils.py:45]: Comment: what is the values of  percents " if is_numeric_dtype(new) & is_numeric_dtype(reference)" return false in get_population_percentages().

File: pulsar_metrics/metrics/utils.py [hunk 3]
```diff
@@ -34,36 +34,35 @@ def get_population_percentages(new: pd.Series, reference: pd.Series, binned: boo
...
 
-    kl_div = entropy(percents["new"], percents["ref"])
-
-    return kl_div
+    return entropy(percents["new"], percents["ref"])
 
 
 def population_stability_index(new: pd.Series, reference: pd.Series, binned: bool = False):
```
Comment (jananjoy) [inline pulsar_metrics/metrics/utils.py:50]: Clarification: If the input type is of pf.Dataframe. Will the same code works? Do you need isinstance() validation before calculation.

File: pulsar_metrics/metrics/utils.py (hunk 1 above)
Comment (rs-aben) [reply pulsar_metrics/metrics/utils.py:37]: I see ! Something needs to be changed here got include categorical variables for both new and reference and a TypeError if not of the same type.

File: pulsar_metrics/metrics/utils.py (hunk 2 above)
Comment (rs-aben) [reply pulsar_metrics/metrics/utils.py:45]: I'll have to correct the get_population_percentages() function as per my comment above

File: pulsar_metrics/metrics/utils.py (hunk 3 above)
Comment (rs-aben) [reply pulsar_metrics/metrics/utils.py:50]: We can add a validation that new and reference are unidimensional. However this function will be changed in the future

**Review by rs-aben — APPROVED** [review]
I have incorporated some of your comments. Thanks !

**Review by rs-aben — APPROVED** [review]
I have incorporated some of your comments

**Pull request merged**